api/db/seeds/data/common/tooling/session-tooling.js

```diff
@@ -614,10 +616,35 @@ function _addCertificationCandidatesToScoSession(
 }
 
 async function _buildOrganizationLearners(databaseBuilder, organizationId, configSession) {
-  return await databaseBuilder
-    .knex('organization-learners')
-    .where({ organizationId })
-    .limit(configSession.learnersToRegisterCount);
+  for (let index = 0; index < configSession.learnersToRegisterCount; index++) {
+    const otherUser = databaseBuilder.factory.buildUser.withRawPassword({
+      firstName: `Élève-${index}`,
+      lastName: `Non certifiable`,
+      email: `user-with-sco-certification_${index}@example.net`,
+      cgu: true,
+      lang: 'fr',
+    });
+
+    databaseBuilder.factory.buildOrganizationLearner({
+      firstName: otherUser.firstName,
+      lastName: otherUser.lastName,
+      sex: 'M',
+      birthdate: '2000-01-01',
+      birthCity: null,
+      birthCityCode: '75115',
+      birthCountryCode: '100',
+      birthProvinceCode: null,
+      division: 'Terminal',
+      isDisabled: false,
+      organizationId,
+      createdAt: new Date(),
+      updatedAt: new Date(),
+    });
+  }
+
+  await databaseBuilder.commit();
+
+  return await databaseBuilder.knex('organization-learners').where({ organizationId });
```

Comment:
question : Nous avions évoqué le fait de passer par des cas d'utilisations (`usecases`), ça n'est pas possible ici ?

Reply:
j'ai l'impression que celui-ci pourrait au moins répondre au "get" des learners créés : api/src/prescription/organization-learner/domain/usecases/find-groups-by-organization.js et au moins assurer que les orgas-learners créés sont visibles via la view-active-orga-learners

Pour le create j'ai vu cela : api/src/prescription/organization-learner/domain/usecases/create-user-and-reconcile-to-organization-learner-from-external-user.js (mais faudra un samlId, mais je sais pas trop si on peut avoir cela en seed, mais ptet qu'un mec d'accès genre @P-Jeremy saura nous aider :) )

Reply:
le await est pas nécessaire :) 

```suggestion
  return databaseBuilder.knex('organization-learners').where({ organizationId });
```